emission/core/wrapper/suggestion_sys.py

```diff
@@ -499,25 +527,25 @@ def calculate_yelp_server_suggestion_nominatim(uuid):
                     #Not sure to include the amount of carbon saved
                     #Still looking to see what to return with this message, because currently my latitude and longitudes are stacked together in one string
                     # insert_into_db(tripDict, i, yelp_suggestion_trips, uuid)
-                    return {'message' : message, 'method': 'bike', 'rating': str(ratings_bus[a]), 'businessid': a}
+                    return {'message' : message, 'method': 'bike', 'rating': str(ratings_bus[a]), 'businessid': a, "tripid": str(cleaned_trips.iloc[i]["_id"])}
                 except ValueError as e:
                     continue
             elif calculate_distance < distance_in_miles and calculate_distance < 1:
                 try: 
                     message = "Why didn't you walk from " + begin_string_address+ " to " + a + " (tap me to view) " + a + \
                     " has better reviews, closer to your original starting point, and has a rating of " + str(ratings_bus[a])
                     # insert_into_db(tripDict, i, yelp_suggestion_trips, uuid)
-                    return {'message' : message, 'method': 'walk', 'rating': str(ratings_bus[a]), 'businessid': a}
+                    return {'message' : message, 'method': 'walk', 'rating': str(ratings_bus[a]), 'businessid': a, "tripid": str(cleaned_trips.iloc[i]["_id"])}
                 except ValueError as e:
                     continue
             elif calculate_distance < distance_in_miles and calculate_distance >= 5 and calculate_distance <= 15:
                 try: 
                     message = "Why didn't you check out public transportation from " + begin_string_address + " to " + a + " (tap me to view) " + a + \
                     " has better reviews, closer to your original starting point, and has a rating of " + str(ratings_bus[a])
                     # insert_into_db(tripDict, i, yelp_suggestion_trips, uuid)
-                    return {'message' : message, 'method': 'public', 'rating': str(ratings_bus[a]), 'businessid': a}
+                    return {'message' : message, 'method': 'public', 'rating': str(ratings_bus[a]), 'businessid': a, "tripid": str(cleaned_trips.iloc[i]["_id"])}
                 except ValueError as e:
                     continue
-    return {'message': "Your endpoint has either been a non-serviceable category or a closeby option.",'method': 'public transportation', 'rating': None, 'businessid': None}
+    return {'message': "Your endpoint has either been a non-serviceable category or a closeby option.",'method': 'public transportation', 'rating': None, 'businessid': None, "tripid": None}
```

Comment:
why is `tripid` `none` here? It's not really an error, right?

Reply:
Changed it.